Gemfile

```diff
@@ -83,7 +83,7 @@ group :test do
   gem 'rspec-collection_matchers'
   gem 'rspec-instafail'
   gem 'rspec-its'
-  gem 'rspec-rails', '~> 6.1.4'
+  gem 'rspec-rails', '~> 7.0.1'
```

Comment:
It appears that there are some breaking changes introduced with this major version release. I will create a ticket to perform a proper code upgrade.